### PR TITLE
add some example to clarify the answer

### PR DESCRIPTION
**Pull request by phamngocquy**

### CODE REVIEW

File: answers/database.md [hunk 1]
```diff
@@ -70,12 +70,14 @@ Indexing adds a data structure with columns for the search conditions and a poin
 
 ## What algorithm and data structure indexing used? And why?
 
-Index can use a hash table or B-Tree.
+Index can use a hash table or B-Tree. [The index decided used depend on data type and how you query data](https://dev.mysql.com/doc/refman/8.0/en/index-btree-hash.html).
```
Comment (tamhoang1412) [inline answers/database.md]: @phamngocquy thank you for your contribution :pray: 
`The index decided used depend on data type and how you query data` It seems like the grammar of this sentence is not correct. Can you double check?

Comment (phamngocquy) [reply answers/database.md]: I just updateed it. plz check

**Review by nvh95 — APPROVED** [review]
LGTM!

**Review by tamhoang1412 — APPROVED** [review]

File: answers/database.md (hunk 1 above)
Comment (tamhoang1412) [reply answers/database.md]: @phamngocquy thank you!

**Comment by nvh95** [comment]
@all-contributors please add @phamngocquy for content

**Comment by nvh95** [comment]
@phamngocquy Thanks for your contributions. Welcome to contributors 🎉.

**Pull request merged**